astropy/utils/console.py

```diff
@@ -735,15 +736,24 @@ def work(i):
             of the chunks of ``items`` that are submitted as separate tasks
             to the process pool.  A large step size may make the job
             complete faster if ``items`` is very long.
+
+        multiprocessing_start_method : str, optional
+            Useful primarily for testing; if in doubt leave it as the default.
+            When using multiprocessing, certain anomalies occur when starting
+            processes with the "spawn" method (the only option on Windows);
+            other anomalies occur with the "fork" method (the default on Unix).
```

Comment:
This isn't quite true anymore in Python 3.8 FYI (MacOS X uses 'spawn' there). Maybe you can change Unix -> Linux (here and in other instances below).